### ISSUE TITLE
KeyError: 'hachoir-parser-1.3.4-py27_0.tar.bz2'

### ISSUE THREAD

**Issue by LawrenceBobrov** [conda/conda#2536]
conda create -n ros --clone nomkl
Error: prefix already exists: /home/lawr/anaconda4/envs/ros
lawr@panchenko:~$ conda create -n ros --clone nomkl
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: ....
src_prefix: '/home/lawr/anaconda4/envs/nomkl'
dst_prefix: '/home/lawr/anaconda4/envs/ros'
Packages: 188
Files: 906
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

```
https://github.com/conda/conda/issues
```

Include the output of the command 'conda info' in your report.

Traceback (most recent call last):
  File "/home/lawr/anaconda4/bin/conda", line 6, in <module>
    sys.exit(main())
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/cli/main.py", line 139, in main
    args_func(args, p)
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/cli/main.py", line 146, in args_func
    args.func(args, p)
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/cli/main_create.py", line 49, in execute
    install.install(args, parser, 'create')
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/cli/install.py", line 247, in install
    clone(args.clone, prefix, json=args.json, quiet=args.quiet, index=index)
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/cli/install.py", line 84, in clone
    quiet=quiet, index=index)
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/misc.py", line 246, in clone_env
    sorted_dists = r.dependency_sort(dists)
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/resolve.py", line 729, in dependency_sort
    depends = lookup(value)
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/resolve.py", line 724, in lookup
    return set(ms.name for ms in self.ms_depends(value + '.tar.bz2'))
  File "/home/lawr/anaconda4/lib/python2.7/site-packages/conda/resolve.py", line 573, in ms_depends
    deps = [MatchSpec(d) for d in self.index[fn].get('depends', [])]
KeyError: 'hachoir-parser-1.3.4-py27_0.tar.bz2'
lawr@panchenko:~$ ^C
lawr@panchenko:~$ conda info
Using Anaconda Cloud api site https://api.anaconda.org
Current conda install:

```
         platform : linux-64
    conda version : 4.0.6
```

  conda-build version : 1.20.3
       python version : 2.7.11.final.0
     requests version : 2.10.0
     root environment : /home/lawr/anaconda4  (writable)
  default environment : /home/lawr/anaconda4
     envs directories : /home/lawr/anaconda4/envs
        package cache : /home/lawr/anaconda4/pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/linux-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/linux-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
    is foreign system : False

**Comment by mcg1969** [comment]
What channel did `hachoir-parser` come from? Can you try adding that channel back into your channel list and trying again?

**Comment by kalefranz** [comment]
Clone was fixed on conda 4.1.  Please update to a recent version of conda.  Latest is conda 4.3.17.
